src/lib/utils/__tests__/classNames.js

```diff
@@ -0,0 +1,34 @@
+import { classNames } from '../classNames';
+
+describe('classNames', () => {
+  it('returns filtered class names', () => {
+    const result = classNames(
+      'class-1',
+      'class-2 class-3',
+      ' ',
+      0,
+      1,
+      null,
+      undefined,
+      true,
+      false,
+    );
+
+    expect(result).toEqual('class-1 class-2 class-3');
```

Comment:
I think we should also test objects like `styles.root` — that's what we actually use the most.

Reply:
See comment above. It is related to loader.